helper/helper.go

```diff
@@ -0,0 +1,81 @@
+package helper
+
+import (
+	"archive/tar"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func GetApplication(applicationsPath, applicationUUID string) (string, string, error) {
```

Comment:
This really seems like something that should live in the app package, surely? Only put stuff in generic `helper` package if it has generic application and wouldn't fit anywhere else (you can have separate code files obv.)